### PR TITLE
look in /etc/ for mime.types + message user if no mime.types is found.

### PR DESCRIPTION
**Pull request by christopher-john-czettel**
see issue  #556.

### CODE REVIEW

**Comment by christopher-john-czettel** [comment]
probably also closes: log when no mime.types file is found #635

File: sendlib.c [hunk 1]
```diff
@@ -956,6 +956,7 @@ int mutt_lookup_mime_type(struct Body *att, const char *path)
   char subtype[STRING], xtype[STRING];
   int szf, sze, cur_sze;
   int type;
+  int found = 0;
```
Comment (toogley) [inline sendlib.c]: i think, this can be changed into a boolean value.

**Comment by christopher-john-czettel** [comment]
@gahr the code below the switch case keeps on searching until all files have been parsed and only remembers the last instance of (mime-)type and xtype.

The order was:
```
/etc/mime.types
~/.mime.types
SYSCONFDIR/mime.types
PKGDATADIR/mime.types
```
Therefore the mime.types in SYSCONF/PKGDATADIR would override any changes in the users ~/.mime.types

I changed to order to the following:
```
/etc/mime.types
SYSCONFDIR/mime.types
PKGDATADIR/mime.types
~/.mime.types
```
It should now work as expected.

**Comment by christopher-john-czettel** [comment]
Although I am not quite sure if this change (although making sense) could break someones config, as now the users .mime.types is higher rated.

**Comment by flatcap** [comment]
I've clang-formatted and squashed (no code changes).

**Pull request closed without merging**